src/models/users/AnonymousUser.php

```diff
@@ -48,7 +48,7 @@ private function fillUserData(): void
         $this->userData['pdf_format'] = 'A4';
         $this->userData['userid'] = 0;
         $this->userData['entrypoint'] = 1;
-        $this->userData['initials'] = '웃';
+        $this->userData['initials'] = '☉'; // generic character utf &#9737. see issue #5778
```

Comment:
why use html encoding here, use unicode value `U+2609`

Reply:
also, no need to mention issue, as the issue in question doesn't inform on why this character was chosen, but rather informs on why a previously chosen character was a wrong choice. If necessary, blame can retrace the history. Also, add comment on top of line, we don't really add inline comments.